File: C++ Programming/Default.cpp
#include<iostream>
using namespace std;

float Calaculate(float Marks, float Outof = 100)
{
    float Percentage = ((Marks / Outof) * 100);
    return Percentage;
}

int main()
{
    float Ans = 0.0f;

    Ans = Calaculate(86,100);   // Calculate(86,100)
    cout<<"Percentage : "<<Ans<<"\n";

    Ans = Calaculate(86);       // Calculate(86)
    cout<<"Percentage : "<<Ans<<"\n";
    
    Ans = Calaculate(320,400);  // Calculate(320,400)
    cout<<"Percentage : "<<Ans<<"\n";
    
    return 0;
}

/*

float Calaculate(float Outof = 100, float Marks)
{

}

*/

File: Typecipt Programmimg/Default.ts

function Percentage(Marks : number, Outof : number = 100) : number
{
    var Result : number = 0;
    Result = ((Marks / Outof) * 100);
    return Result;
}

var Ans : number = 0;

Ans = Percentage(89,200);
console.log("Percentage is : "+Ans);

Ans = Percentage(89);
console.log("Percentage is : "+Ans);
